css/main.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&family=Roboto:wght@100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: 'Inter', sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.headline-sec .profession {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.job {
+  margin-top: 3rem;
+}
+
+.title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+}
+
+.nav-toggle {
+  background: none;
+  border: none;
+  cursor: pointer;
+  right: 1em;
+  top: 1em;
+}
+
+.header .welcome-logo {
+  font-size: 1.25rem;
+  font-weight: 600;
+}
+
+.name {
+  margin-top: 10px;
+  color: #172b4d;
+}
+
+.job-title {
+  color: #36b37e;
+}
+
+.text {
+  font-weight: 400;
```

Comment:
- [ ] Please make that your text section has the right margins to the left and the right.
## Your section 👇️
![Screenshot from 2022-09-21 19-43-26](https://user-images.githubusercontent.com/89228743/191596127-d31a5d3a-5e34-4c38-a35d-401810be2ec1.png)

## the figma design 👇️

![Screenshot from 2022-09-21 19-42-17](https://user-images.githubusercontent.com/89228743/191595911-8841ee5e-382e-470e-8df2-48a3291ec6de.png)